cmd/build/v2/services.go

```diff
@@ -118,3 +118,24 @@ func (bc *OktetoBuilder) checkImageAtGlobalAndSetEnvs(service string, options *t
 	return true, nil
 
 }
+
+// GetServicesToBuildFromSubset returns the services it has to built because they are not already built from a subset of services
+func (bc *OktetoBuilder) GetServicesToBuildFromSubset(ctx context.Context, manifest *model.Manifest, subset []string) ([]string, error) {
+	for name := range manifest.Build {
+		needsBuild := false
+		for _, toBuildName := range subset {
+			if name == toBuildName {
+				needsBuild = true
+				break
+			}
+		}
+		if !needsBuild {
+			delete(manifest.Build, name)
```

Comment:
Is it safe to manipulate the manifest removing things?

Reply:
We should be ignoring them, I don't like to delete them so I'll check if I can do it in another way so we don't delete them